### ISSUE TITLE
Can't run Zonation

### ISSUE THREAD

**Issue by LaureVelez** [wkmor1/rzonation#4]
Hello,

I can not run zonation on my computer
macOS Catalina version 10.15.6
R version is  4.0.2

I tried to run the exemple 
library(rzonation)

r1 <- raster(matrix(runif(200^2, 0, 1), 200))

r2 <- raster(matrix(runif(200^2, 0, 1), 200))

plan <- zonation(stack(r1, r2))

and I got an error
Error in zonation(features = feature_files, params, settings, dir, alpha,  : 
  zonation binary not found

I guess that the core of zonation might not be installed
How do I have to proceed? Is it working on Mac?

Thanks,

**Comment by wkmor1** [comment]
Yeah, rzonation does not include the Zonation software itself. You need to install that independently. Not sure it can be run on macOS natively at this point. But it can be run through docker. e.g., https://github.com/wkmor1/build-zig4/blob/master/Dockerfile
